sites/kit.svelte.dev/package.json

```
@@ -14,23 +14,24 @@
 		"@sveltejs/adapter-vercel": "workspace:^",
 		"@sveltejs/amp": "workspace:^",
 		"@sveltejs/kit": "workspace:^",
-		"@sveltejs/site-kit": "6.0.0-next.36",
-		"@types/d3-geo": "^3.0.3",
-		"@types/node": "^20.4.9",
+		"@sveltejs/site-kit": "6.0.0-next.42",
+		"@types/d3-geo": "^3.0.4",
+		"@types/node": "^20.6.0",
```

does it work to keep it in sync with kit itself?
```suggestion
		"@types/node": "^16.18.6",
```